examples/sentiment/train_bert.py

```diff
@@ -166,21 +206,23 @@ def main():
         bert_model=BERT_MODEL_NAME,
         n_classes=num_labels,
         n_warmup_steps=n_warmup_steps,
-        n_training_steps=n_training_steps
+        n_training_steps=n_training_steps,
     )
```

Comment:
This comma is needed to prevent unexpected errors in the future, it prevents string concatenation if you accidentally forget the comma. The comma is added because [black](https://github.com/psf/black) formatter. See more https://www.youtube.com/shorts/3QQDaqpNb1Y

Reply:
@hungpham3112 The arguments are not string literals so there should not be any risks of concatenation. Black style seems nice, but also a superset of PEP8 that has not been officially endorsed by the Python dev team.

underthesea can keep this in mind of the future regarding style checking.